### PR TITLE
Secure Practice MailRisk Data Connector KeyVault

### PR DESCRIPTION
**Pull request by ingin97**
Required items, please complete
   
   Change(s):
   - Properly fetch values from KeyVault using environment variables
   - NB: The short url https://aka.ms/sentinel-mailrisk-functionapp should be changed to pointing to the zip package in master in this repo after this PR is merged.

   Reason for Change(s):
   - Key vaults secrets not properly fetched

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by v-atulyadav** [comment]
Looking into this

**Comment by v-vdixit** [comment]
We are looking into this, thanks!

**Comment by v-vdixit** [comment]
We are discussing this with the team, will update shortly, thanks!

**Comment by ingin97** [comment]
@v-vdixit Anything I can do to help?

**Comment by v-vdixit** [comment]
We are looking into this, thanks!

File: Solutions/MailRisk/Data Connectors/MailRiskSentinelIntegration.zip
Comment (v-vdixit) [inline Solutions/MailRisk/Data Connectors/MailRiskSentinelIntegration.zip:1]: Hi @ingin97 we can see that the zip files contain a folder which further contains the required files, please ensure that all related files are directly added in the main folder, thanks!

Comment (ingin97) [reply Solutions/MailRisk/Data Connectors/MailRiskSentinelIntegration.zip:1]: @v-vdixit Oh, my bad. Should be fixed now!

**Comment by v-vdixit** [comment]
Hi @ingin97 we have deployed the function app but since we do not have the required credentials, could you please add screenshots demonstrating successful working of the function app, thanks!

**Comment by ingin97** [comment]
@v-vdixit Hi, here are the screenshot of deployment and successful working of the function app:
image
image

**Comment by v-vdixit** [comment]
Hi @ingin97 Can you please add screenshots for invocation details too by clicking on the date mentioned in these results, so that we can ensure there are no warnings while running the function app, thanks!

**Comment by v-vdixit** [comment]
Hi @ingin97 we are waiting for your update, thanks!

**Comment by ingin97** [comment]
Hi @v-vdixit Here is a screenshot with invocation details for one run containing emails to push and one without.
image

**Review by v-vdixit — APPROVED** [review]

**Pull request merged**